package.json

```
@@ -0,0 +1,44 @@
+{
+  "name": "request-network-template",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@requestnetwork/create-request-form": "^0.1.0",
+    "@requestnetwork/invoice-dashboard": "^0.1.0",
+    "@requestnetwork/request-client.js": "0.45.0",
+    "@requestnetwork/shared": "^0.1.0",
+    "@requestnetwork/web3-signature": "0.5.0",
+    "@web3-onboard/coinbase": "^2.2.7",
+    "@web3-onboard/core": "^2.21.6",
+    "@web3-onboard/fortmatic": "^2.0.19",
+    "@web3-onboard/gnosis": "^2.2.2",
+    "@web3-onboard/infinity-wallet": "^2.0.4",
+    "@web3-onboard/injected-wallets": "^2.10.16",
+    "@web3-onboard/ledger": "^2.6.0",
+    "@web3-onboard/react": "^2.8.17",
+    "@web3-onboard/trezor": "^2.4.4",
+    "@web3-onboard/trust": "^2.0.4",
+    "@web3-onboard/walletconnect": "^2.5.5",
```

Please remove unused packages.
```suggestion
    "@web3-onboard/coinbase": "^2.2.7",
    "@web3-onboard/core": "^2.21.6",
    "@web3-onboard/gnosis": "^2.2.2",
    "@web3-onboard/injected-wallets": "^2.10.16",
    "@web3-onboard/ledger": "^2.6.0",
    "@web3-onboard/react": "^2.8.17",
    "@web3-onboard/trezor": "^2.4.4",
    "@web3-onboard/trust": "^2.0.4",
    "@web3-onboard/walletconnect": "^2.5.5",
```